atkinson/errors/reporters/__init__.py

```diff
@@ -12,22 +12,41 @@ class BaseReport(ABC):
     @classmethod
     @abstractmethod
     def new(cls, title, description, config):
-        """ Create a new report and return a report object """
+        """
+        Create a new report and return a report object
+
+        :param title: A title for the report (str)
```

Comment:
Just thought - don't we use title to return a report_id?  Maybe the description should mention that this title uniquely identifies the report?